### PR TITLE
Pre release commit

### PR DESCRIPTION
**Pull request by wxtim**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ below:
 * Bruno P. Kinoshita (National Institute of Water and Atmospheric Research, New Zealand)
 * Tim Pillinger (Met Office, UK)
 * Ronnie Dutta (Met Office, UK)
+* Mel Hall (Met Office UK)
```
Comment (wxtim) [inline CONTRIBUTING.md:50]: Additionally `git shortlog -sn` suggests that the following names may also be missing·..
- Declan Valters
- Jonny Williams
- Paul Cresswell
- Roddy Sharp

File: CHANGES.md [hunk 2]
```diff
@@ -3,6 +3,19 @@
 Go to https://github.com/metomi/rose/milestones?state=closed
 for a full listing of issues for each release.
 
+## 2019.01.4 (2019-12-xx)
```
Comment (oliver-sanders) [inline CHANGES.md:6]: Can make this today's date

File: CHANGES.md [hunk 3]
```diff
@@ -3,6 +3,19 @@
...
+
+### Noteworthy Changes
+[#2440](https://github.com/metomi/rose/pull/2440) Fix bug where incremental
+fileinstall did not work if a file was installed previously, overwritten, then
+returned to to the orginal source.
+
+
 --------------------------------------------------------------------------------
 
 ## 2019.01.4 (2019-12-xx)
```
Comment (oliver-sanders) [inline CHANGES.md:21]: 🍌  can we back-fill the last release date.

File: CONTRIBUTING.md (hunk 1 above)
Comment (oliver-sanders) [reply CONTRIBUTING.md:50]: @datamel can you ok this (since it means you agree to the CLA).

Comment (oliver-sanders) [reply CONTRIBUTING.md:50]: The other contributors predate the current CLA, we will have to poke them to add themselves if/when they next contribute.

File: CHANGES.md (hunk 3 above)
Comment (wxtim) [reply CHANGES.md]: Yes/

File: CHANGES.md (hunk 2 above)
Comment (wxtim) [reply CHANGES.md]: Yes.

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**